atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/timestamp/CleanTransactionRange.java

```diff
@@ -55,17 +70,46 @@ protected boolean requireTimestamp() {
     protected int executeTimestampCommand(AtlasDbServices services) {
         KeyValueService kvs = services.getKeyValueService();
 
+        byte[] startBytes = TransactionConstants.getValueForTimestamp(startTimestamp);
```

Comment:
What's the behavior if `startTimestamp` isn't specified? There's a null check in the `kvs.getRange` method below, but this doesn't look safe. 

Can you explain the logic behind checking timestamp length equality?

Reply:
It's because of var longs. same length == we can reduce the size of our range scan to limit it to just that "length"